packages/protoc/package-lock.json

```diff
@@ -1,5 +1,599 @@
 {
   "name": "@protobuf-ts/protoc",
   "version": "2.8.3",
-  "lockfileVersion": 1
+  "lockfileVersion": 3,
```

Comment:
I'm not sure this will work. CI runs with Node 14.5 ([see here](https://github.com/timostamm/protobuf-ts/blob/3a5b9f982d30d3422d856176d822bed027e981e4/.circleci/config.yml#L7)), and I don't know at the top of my head if it handles lock file version 3 correctly. Could you use nvm or fnm to switch to the old Node version, and re-create the lock file?